janitor/functions/min_max_scale.py

```diff
@@ -8,11 +10,12 @@
 @deprecated_alias(col_name="column_name")
 def min_max_scale(
     df: pd.DataFrame,
-    old_min=None,
-    old_max=None,
-    column_name=None,
-    new_min=0,
-    new_max=1,
+    feature_range: tuple[int | float, int | float] = (0, 1),
+    column_name: str
+    | int
+    | list[str | int]
+    | tuple[str | int]
+    | pd.Index = None,
```

Comment:
I thought there should be a _typing.py script that could contain some common typing such as `column_name`.